README.md

```diff
@@ -17,6 +19,11 @@ gcc --version
 ```
 python3 --version
 ```
+* make python3 your default python version (needed by the next pip command)
+```
+virtualenv -p python3 python3.venv
+source python3.venv/bin/activate
+```
 * python requirements
 ```
 pip install -r requirements.txt
```

Comment:
`virtualenv`s are good but I don't think we need to force this in this case.  A simple `pip3 install` would also work. (My two cents).